dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/transport/AbstractClient.java

```diff
@@ -55,13 +63,23 @@ public abstract class AbstractClient extends AbstractEndpoint implements Client
 
     protected volatile ExecutorService executor;
 
+    protected volatile ScheduledExecutorService connectivityExecutor;
+
+    private ApplicationModel applicationModel;
+
+    protected long reconnectDuaration;
+
     public AbstractClient(URL url, ChannelHandler handler) throws RemotingException {
         super(url, handler);
         // set default needReconnect true when channel is not connected
         needReconnect = url.getParameter(Constants.SEND_RECONNECT_KEY, true);
 
+        applicationModel = url.getOrDefaultApplicationModel();
```

Comment:
Directly get Framework Model here would be better

Reply:
@AlbumenJ 
Thank you for your review. 
I've made the adjustments based on your suggestions. Could you please check if it aligns with your expectations?